app/assets/stylesheets/thredded/components/_currently-online.scss

```diff
@@ -1,4 +1,8 @@
-.currently-online {
+.thredded--currently-online {
+  @include media($mobile) {
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets